v1.1/admin-ui-jobs-page.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Jobs Page
+toc: false
+---
+
+The **Jobs** page of the Admin UI provides details of the backup and restore jobs, as well as schema changes performed across all nodes in the cluster.
```

Comment:
nit: details about backup/restore jobs as well as schema changes performed across all nodes in the cluster.